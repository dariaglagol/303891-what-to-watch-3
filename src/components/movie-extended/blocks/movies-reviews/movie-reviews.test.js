import React from "react";
import renderer from "react-test-renderer";
import MovieReviews from "./movies-reviews";

const MovieReviewsMock = [
  {
    text: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director&apos;s funniest and most exquisitely designed movies in years.`,
    author: `Kate Muir`,
    date: `December 24, 2016`,
    rating: 8.9
  }, {
    text: `Anderson&apos;s films are too precious for some, but for those of us willing to lose ourselves in them, they&apos;re a delight. &quot;The Grand Budapest Hotel&quot; is no different, except that he has added a hint of gravitas to the mix, improving the recipe.`,
    author: `Bill Goodykoontz`,
    date: `November 18, 2015`,
    rating: 8.0
  }
];

it(`Render movie reviews`, () => {
  const movieReviewsComponent = renderer
    .create(
        <MovieReviews
          reviews={MovieReviewsMock}
        />
    ).toJSON();

  expect(movieReviewsComponent).toMatchSnapshot();
});
