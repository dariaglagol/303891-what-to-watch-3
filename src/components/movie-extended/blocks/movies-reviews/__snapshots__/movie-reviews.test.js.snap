// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render movie reviews 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            author
          </cite>
          <time
            className="review__date"
            dateTime="2016-12-24"
          >
            date
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        8.9
      </div>
    </div>
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          text
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            author 1
          </cite>
          <time
            className="review__date"
            dateTime="2016-12-24"
          >
            date 2
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        8
      </div>
    </div>
  </div>
</div>
`;
